static/javascript/launch.js

```diff
@@ -584,9 +591,11 @@ function retrieveResponseId() {
   let responseIdButton = document.querySelector("#response-id-btn");
 
   if (responseId) {
-    responseIdButton.classList.remove("btn--hidden");
+    responseIdButton.classList.remove("ons-btn--disabled");
+    responseIdButton.removeAttribute("disabled");
   } else {
-    responseIdButton.classList.add("btn--hidden");
+    responseIdButton.classList.add("ons-btn--disabled");
+    responseIdButton.setAttribute("disabled");
```

Comment:
This cannot be done using `setAttribute()` as it always requires two attributes, name and value. Consider using `disabled = boolean` as described in: https://www.w3schools.com/jsref/prop_pushbutton_disabled.asp.

Reply:
Updated 👍